docs/manual/selector.md

```diff
@@ -1,10 +1,41 @@
-# Selector
-Selector nodes will attempt to execute each of its children and reports `SUCCESS` status code in case one of the children reports a `SUCCESS` status code. In case all children report a `FAILURE` status code, this node will also return `FAILURE` status code. This node will attempt to process all its children every single tick, even if one of them is currently `RUNNING` already.
+# Selector Node
+The Selector node is another fundamental building block in Behavior Trees, used to manage decision-making among multiple child nodes. It helps you define different behaviors for your game characters or objects based on varying conditions.
+
+## How does it work?
+The Selector node tries to execute each of its children one by one, in the order they are connected. It reports a `SUCCESS` status code if any child reports a `SUCCESS`. If all children report a `FAILURE` status code, the Selector node also returns `FAILURE`.
```

Comment:
I'd add a small line explaining what reporting SUCCESS and FAILURE would do, like "If a child reports a `SUCCESS`, the selector will stop executing."